### ISSUE TITLE
@here metion is not happening & Succeeded and Failure icons are also missing

### ISSUE THREAD

**Issue by shridharkalagi** [8398a7/action-slack#93]
**Describe the bug**
@here metion is not happening & Succeeded and Failure icons are also missing

**Screenshots**
@here mention is not working
![image](https://user-images.githubusercontent.com/18380219/88459680-c4d5c300-ceb4-11ea-9817-0b80a29bb78f.png)

Succeeded and Failed icons are missing
![image](https://user-images.githubusercontent.com/18380219/88459683-d1f2b200-ceb4-11ea-86c1-add8c8dc6b08.png)

Here is the link to actions.yml file - [https://github.com/umapps/UMApps-Test/blob/master/.github/workflows/maven.yml#L27](https://github.com/umapps/UMApps-Test/blob/master/.github/workflows/maven.yml#L27)

**Comment by 8398a7** [comment]
`@here` Mentions not working seems to be a glitch. Thanks for the report.

The icons are specifics.
You have specified text, so they have been overwritten.
refs: https://action-slack.netlify.app/migration/01-migrate-from-v2-to-v3#overwritingtext

It's only mentioned in the migrate guide, which is unfriendly and will be corrected.

```yaml
        with:
          status: ${{ job.status }}
          fields: repo,message,commit,author,action,eventName,ref,took
          text: API Tests # this
```

**Comment by 8398a7** [comment]
Fixed a bug in v3.5.2 where `here` mention was not working.
Thanks for the report.

**Comment by shridharkalagi** [comment]
Thank you
